.goreleaser.yaml

```diff
@@ -29,7 +29,11 @@ builds:
         goarch: arm
     dir: src
     ldflags:
-      - -X github.com/f1bonacc1/process-compose/src/config.Version={{.Tag}} -X github.com/f1bonacc1/process-compose/src/config.CheckForUpdates=true -s -w
+      - -X github.com/f1bonacc1/process-compose/src/config.Version={{.Tag}}
+      - -X github.com/f1bonacc1/process-compose/src/config.CheckForUpdates=true
+      - -X github.com/f1bonacc1/process-compose/src/config.Commit={{.ShortCommit}}
+      - -X github.com/f1bonacc1/process-compose/src/config.Date={{.Date}}
```

Comment:
This is a misnaming, dates are generally referring to days, instead of timestamps when the build happened, which is the actual case.

Reply:
I can change it to:
`github.com/f1bonacc1/process-compose/src/config.Date={{.CommitDate}}`
Go releaser support that out of the box (nice) and returns `2022-12-24T21:20:48Z`
And in Makefile I change it to:
`DATE       ?= $(shell TZ=UTC0 git show --quiet --date='format-local:%Y-%m-%dT%H:%M:%SZ' --format="%cd")`
Which also returns the last commit timestamp in the same format: `2022-12-24T21:20:48Z`

It's also somewhat supported with flake:
`self.lastModifiedDate`
The format is a little bit different, but also in UTC: `20221224212048`

However no matter what I tried I can't make a short commit work, it always returns dirty. Even after taking your changes.

Reply:
If you have a clean git status, it works properly:
```
➜  process-compose git:(version_info_upstream) git status                           
On branch version_info_upstream
nothing to commit, working tree clean
➜  process-compose git:(version_info_upstream) nix build .                         
➜  process-compose git:(version_info_upstream) ./result/bin/process-compose version
Process Compose
Version:        v0.29.1
Commit:         89254f1
Date (UTC):     20221225200050
License:        Apache-2.0

Written by Eugene Berger
```
I haven't found a nice and easy way to parse the `lastModifiedDate` into something a bit more `date` like without too much hassle. IMO the way this PR stands in fine for now.